ocs_ci/helpers/helpers.py

```diff
@@ -735,14 +735,14 @@ def create_multiple_pvcs(
     return ocs_objs, tmpdir
 
 
-def delete_bulk_pvcs(pvc_yaml_dir, pv_names_list):
+def delete_bulk_pvcs(pvc_yaml_dir, pv_names_list, namespace):
```

Comment:
Hope you checked and updated all the TCs which calls this function with additional required namespace param

Reply:
Hi Ram, Only this test case was using delete_bulk_pvcs function from helpers.py